src/core_plugins/tag_cloud/public/cloud_vis_params.html

```diff
@@ -0,0 +1,28 @@
+<div class="form-group">
+
+  <div class="form-group">
+    <label>Text Scale</label>
+    <select ng-model="vis.params.textScale" class="form-control">
+      <option value="linear" selected="selected">linear</option>
+      <option value="log">log</option>
+      <option value="sqrt">square root</option>
```

Comment:
I think options should be defined in cloud.js. Other visualizations do the same (check kbn_vislib_vis_types (map specifically uses different labels/values)) and I think its cleaner than defining them in template files.